examples/nft/scripts/transfer/common.js

```diff
@@ -26,10 +27,17 @@ exports.printLocalNFT = async function (deployer, account, appId) {
     try {
         const localState = await readLocalStateSSC(deployer, account, appId);
         // each nft is stored as a one record in user store
-        let count = 0;
-        if (localState !== undefined)
-            count = localState.length
-        console.log('NFT balance of', account, ':', count);
+        let holdings = "";
+        if (localState === undefined) {
+            holdings = "none";
+        }
+        else {
+            for(const l of localState) {
+                const key = decode(Buffer.from(l.key, 'base64'));
+                holdings += key.toString() + ' ';
```

Comment:
`holdings` should be an array. Avoid using string concatenation in a loop. Instead use `array.push` and after the loop you can use `array.join` to create a string.

Reply:
ok :+1: